recipes/ocaml/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ocaml" %}
+{% set version = "4.06.0" %}
```

Comment:
you can use version_major and version_minor here if you like.

Reply:
Like in https://github.com/conda-forge/pygit2-feedstock/blob/2559bf9dce47390dff9aed0ce3562a16a6bda1d7/recipe/meta.yaml - good idea, thanks!